packages/app/src/components/nominationBanner/nominationBanner.js

```diff
@@ -17,9 +18,16 @@ const NominationBanner = ({ nomination }) => {
   const minutes = newDate.toLocaleTimeString([], { hour: '2-digit', minute: '2-digit' });
   const finalDate = `${time} – ${minutes}`;
   const [activeNomination, setActiveNomination] = useContext(ActiveNominationContext);
-  const declineStatus = 'Declined'
+  
+
+  const [modalState, setModalState] = useState(false);
```

Comment:
Maybe call this `isModalVisible` so the intent is more clear.

Reply:
Done!